### PR TITLE
HBASE-22318 Fix for warning The POM for org.glassfish.javax.el is missing

### PR DESCRIPTION
**Pull request by dbist**

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/PreCommit-HBASE-CONNECTORS-Build/35/

**Comment by busbey** [comment]
hurm. the buildbot doesn't have any logs?

**Comment by busbey** [comment]
no logs for mvn install in any case. let me try redoing the build with debug on.

**Comment by busbey** [comment]
okay looks like rebuilds don't work. I've changed the default to be debug output.

**Comment by busbey** [comment]
oh wait does this repo use keyword checking?

**Comment by busbey** [comment]
retest build

**Comment by busbey** [comment]
it does! excellent.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/PreCommit-HBASE-CONNECTORS-Build/38/

**Comment by busbey** [comment]
@petersomogyi when you have a minute, it looks like we're not actually running any maven commands

e.g.
```

11:14:19 ============================================================================
11:14:19 ============================================================================
11:14:19                            maven install: master
11:14:19 ============================================================================
11:14:19 ============================================================================
11:14:19 
11:14:19 
11:14:19 [Fri Apr 26 16:14:19 UTC 2019 DEBUG]: MultiJDK not configured.
11:14:19 [Fri Apr 26 16:14:19 UTC 2019 DEBUG]: MultiJDK not configured.
11:14:19 [Fri Apr 26 16:14:19 UTC 2019 DEBUG]: Start clock
11:14:19 [Fri Apr 26 16:14:19 UTC 2019 DEBUG]: offset clock by 0
11:14:19 [Fri Apr 26 16:14:19 UTC 2019 DEBUG]: Stop clock
11:14:19 [Fri Apr 26 16:14:19 UTC 2019 DEBUG]: add_vote_table +1 mvninstall 0 master passed
11:14:19 [Fri Apr 26 16:14:19 UTC 2019 DEBUG]: Starting archiving process
11:14:19 [Fri Apr 26 16:14:19 UTC 2019 DEBUG]: Ending archiving process
11:14:19 [Fri Apr 26 16:14:19 UTC 2019 DEBUG]: Running jshint_precompile
11:14:19 [Fri Apr 26 16:14:19 UTC 2019 DEBUG]: Starting archiving process
11:14:19 [Fri Apr 26 16:14:19 UTC 2019 DEBUG]: Ending archiving process
```

**Comment by petersomogyi** [comment]
My first guess is the personality file is incorrect which was added recently in PR23.

**Comment by busbey** [comment]
excellent guess!

```

# customizing yetus build to run scaladoc plugin only on hbase-spark project
function personality_modules
{
  local testtype="$2"

  if [[ "$testtype" != "scaladoc" ]]; then
    return
  fi
```

this says "if you're doing something other than scaladoc there's no work to do." let me get a PR up

**Comment by busbey** [comment]
@dbist can you rebase onto the latest master branch? the CI bot should properly run tests now.

**Comment by petersomogyi** [comment]
retest build

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/PreCommit-HBASE-CONNECTORS-Build/41/

File: pom.xml [hunk 1]
```diff
@@ -139,6 +139,7 @@
          comes in via hbase-thirdparty hbase-shaded-netty-->
     <netty.hadoop.version>3.6.2.Final</netty.hadoop.version>
     <os.maven.version>1.6.1</os.maven.version>
+    <glassfish.el.version>3.0.1-b08</glassfish.el.version>
```
Comment (busbey) [inline pom.xml]: how'd you come up with `3.0.1-b08` here? looking at the mvn install run from before the patch it looks like we end up getting `3.0.1-b11`.

Comment (dbist) [reply pom.xml]: I took it from hbase repo, I believe I first tested with b11, it failed for a reason I don't recall. I can try again.

Comment (busbey) [reply pom.xml]: I'm happy to match the hbase repo. :)

**Pull request merged**